keyboards/skeletonkbd/frost68/info.json

```diff
@@ -0,0 +1,124 @@
+{
+    "manufacturer": "skeletonkbd",
+    "keyboard_name": "Frost68",
+    "maintainer": "skeletonkbd",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+	"matrix_pins": {
+		"cols": [
+			"GP21",
+			"GP22",
+			"GP23",
+			"GP24",
+			"GP25",
+			"GP26",
+			"GP28",
+			"GP3",
+			"GP4",
+			"GP5",
+			"GP6",
+			"GP9",
+			"GP10",
+			"GP11",
+			"GP12",
+			"GP13"
+
+		],
+		"rows": [
+			"GP29",
+			"GP27",
+			"GP20",
+			"GP19",
+			"GP18"
+		]
+	},
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+		"vid": "0x1209",
+		"pid": "0x68A0"
+    },
+    "layouts": {
+        "LAYOUT_alice": {
+            "layout": [
+				{"matrix":[0,0], "x":1.76, "y":1.0},
+				{"matrix":[0,1], "x":2.76, "y":1.0},
+				{"matrix":[0,2], "x":3.76, "y":0.92},
+				{"matrix":[0,3], "x":4.9, "y":6.0},
+				{"matrix":[0,4], "x":5.9, "y":6.0},
+				{"matrix":[0,5], "x":6.9, "y":6.0},
+				{"matrix":[0,6], "x":7.9, "y":6.0},
+				{"matrix":[0,7], "x":8.35, "y":14.6},
+				{"matrix":[0,8], "x":9.35, "y":14.6},
+				{"matrix":[0,9], "x":10.35, "y":14.6},
+				{"matrix":[0,10], "x":11.35, "y":14.6},
+				{"matrix":[0,11], "x":13.18, "y":0.9600000000000001},
```

Comment:
Same thing about precision here (and a couple keys down the file)

Reply:
round off coordinates
cbb27dfaf7060c7bc5c895d34456eb506e82b8b8